django_project/users/templates/users/password_reset_complete.html

```diff
@@ -0,0 +1,28 @@
+{% extends "ta_system/base.html" %}
+{% load crispy_forms_tags %}
+{% block content %}
+    <div class="alert alert-info">
+        Your password has been set.
+    </div>
+    <div class='content-section'>
+        <form method='POST'>
```

Comment:
Have you tested logging in with the new password from this page? I'm concerned that since we aren't explicitly setting the endpoint to be views.login() for this form, the POST request will be made by default to the built-in password_rest_complete view.

Reply:
It's working with the new password